content/en/docs/ibm/create-cluster.md

```diff
@@ -62,20 +72,36 @@ export WORKER_NODE_PROVIDER=classic
-- `WORKER_NODE_PROVIDER` specifies the kind of IBM Cloud infrastructure on which the Kubernetes worker nodes will be created. The `classic` type supports worker nodes with GPUs. There are other worker nodes providers including `vpc-classic` and `vpc-gen2` where zone names and worker flavors will be different. Please use `ibmcloud ks zones --provider ${WORKER_NODE_PROVIDER}` to list zone names if using other providers and set the `CLUSTER_ZONE` accordingly.
+where:
+
+- `KUBERNETES_VERSION` specifies the Kubernetes version for the cluster. Run `ibmcloud ks versions` to see the supported
+  Kubernetes versions. If this environment variable is not set, the cluster will be created with the default version set
+  by IBM Cloud Kubernetes Service. Refer to
+  [Minimum system requirements](https://www.kubeflow.org/docs/started/k8s/overview/#minimum-system-requirements)
+  and choose a Kubernetes version compatible with the Kubeflow release to be deployed.
+- `CLUSTER_ZONE` identifies the regions or location where cluster will be created. Run `ibmcloud ks locations` to
+  list supported IBM Cloud Kubernetes Service locations. For example, choose `dal13` to create your cluster in the
+  Dallas (US) data center.
+- `WORKER_NODE_PROVIDER` specifies the kind of IBM Cloud infrastructure on which the Kubernetes worker nodes will be
+  created. The `classic` type supports worker nodes with GPUs. There are other worker nodes providers including
+  `vpc-classic` and `vpc-gen2` where zone names and worker flavors will be different. Please use
+  `ibmcloud ks zones --provider classic` to list zone names for `classic` provider and set the `CLUSTER_ZONE`
+  accordingly.
 - `CLUSTER_NAME` must be lowercase and unique among any other Kubernetes
   clusters in the specified `${CLUSTER_ZONE}`.
 
-**Notice**: If choosing other Kubernetes worker nodes providers than `classic`, refer to the IBM Cloud official document [Creating clusters](https://cloud.ibm.com/docs/containers?topic=containers-clusters) for detailed steps.
+**Notice**: Refer to [Creating clusters](https://cloud.ibm.com/docs/containers?topic=containers-clusters) in the IBM
+Cloud documentation for additional information on how to set up other providers and zones in your cluster.
 
 ### Choosing a worker node flavor
 
-The worker nodes flavor name varies from zones and providers. Run `ibmcloud ks flavors --zone ${CLUSTER_ZONE} --provider ${WORKER_NODE_PROVIDER}` to list available flavors. For example, following are some flavors supported in the `dal13` zone with `classic` worker node provider.
+The worker nodes flavor name varies from zones and providers. Run 
+`ibmcloud ks flavors --zone ${CLUSTER_ZONE} --provider ${WORKER_NODE_PROVIDER}` to list available flavors.
 
-```text
-$ ibmcloud ks flavors --zone dal13 --provider classic
+For example, following are some flavors supported in the `dal13` zone with `classic` node provider.
+
+```shell
+ibmcloud ks flavors --zone dal13 --provider classic
 OK
```

Comment:
[three back ticks to close the `ibmcloud ks....` code block]

Example output:

[three back ticks]
OK
...
...
[three back ticks to close the text block]